### ISSUE TITLE
Highlight priority datasets for curators

### ISSUE THREAD

**Issue by ryscher** [datadryad/dryad-product-roadmap#3549]
GREI Y3 1.2

In the admin dashboard, highlight items that are in the queue (submitted), which are either associated with an accepted manuscript, OR came out of PPR from an integrated journal.

**Comment by ahamelers** [comment]
The curators have mentioned they would like some kind of tagging option for the admin dashboard, but they haven't quite worked out what they want yet. I think this should be related/part of that, and should be discussed in a dev/curator meeting

**Comment by mahirst** [comment]
We've decided to call them flags, and we will use a color scheme.

- priority - red
- black list - black
- pub ethics - purple
- pending action -yellow
- integrated title - orange
- user testing - gray

These will come later (names to be changed, too, I'm sure):

- (automated) institutions such as UCSB - green
- (automated) list of certain users to look out for for certain reasons - blue

**Comment by jeremywalter** [comment]
possible black list texts (add yours!):
help desk referral
historical

**Comment by lauradryad** [comment]
how about these?

Priority - red
Flagged dataset - black [this will catch everything that isn't automatically flagged with the institution and submitter below]
Sensitive data - purple
Pending action -yellow
Integrated title - orange
User tester - gray
Flagged institution - green
Flagged submitter - blue

**Comment by ahamelers** [comment]
From this list I'm concerned that I haven't been clear enough on how this will work, so I've only implemented 'Priority' dataset tagging in addition to the tagged accounts. We will put it on our dev site to demo and test, and we can talk about what else should be added once we're all really clear on the functionality.

**Comment by jleighherzog** [comment]
@lauradryad : I have a few questions in this list for you. And, I've suggested a flag for technical issues at the bottom. 

Priority - red
Flagged dataset - black [this will catch everything that isn't automatically flagged with the institution and submitter below]
Sensitive data - purple
Pending action - yellow 
Integrated title - orange  (JH: can this be automatically added to datasets)? 
User tester - gray (JH: is this needed)? 
Flagged institution - green
Flagged submitter - blue
Technical - pink (JH: captures bugs, dev escalated issues, etc.; different from 'pending action flag' which might be used to capture escalations within the team or author reply, journal response, etc.)

**Comment by mahirst** [comment]
@jleighherzog I think you were the one who wanted "user tester" as a flag. I'm ok with removing it. We can shorten the names to remove "Flagged", which will also standardize the naming conventions used and satisfy Audrey's requests for shorter names. As for integrated title, that's a dev question but I think it can be done. :)

@lauradryad Curious about the definition of the black flag- it was originally for blacklisting, but what else is it supposed to capture now? I feel like I missed a conversation, maybe when I was on vacation?

**Comment by lauradryad** [comment]
@mahirst @jleighherzog the black flag was meant to capture anything that wasn't already specified. We decided not to do a blacklist because 1) we can flag these people using the Submitter flag and writing a note that explains why and 2) it doesn't look great. I was envisioning this for a dataset specifically that is difficult or requires additional care but doesn't fall into any of the other categories. 

For the integrated ones, are you wanting to tag those for which we receive metadata specifically? The DPC being paid is now a column in the admin dashboard so we don't need it for that. I don't see a strong need for this one, but happy to have if you want it. 

Priority - red
Flagged dataset - black [this will catch everything that isn't automatically flagged with the institution and submitter below]
Sensitive data - purple (HSI, endangered species, or anything else that needs a more thorough review)
Pending action - yellow (waiting for author, journal, cxore - any other party except dev - to proceed with dataset)
Integrated - orange (not sure)
Institution - green (done through User page - to flag institutions like UCSB which need additional actions taken)
Submitter - blue (done through User page)
Technical - pink (JH: captures bugs, dev escalated issues, etc.; different from 'pending action flag' which might be used to capture escalations within the team or author reply, journal response, etc.)

Happy to remove the second word for the ones that currently have 2 if that's our standardization.

**Comment by ahamelers** [comment]
> Integrated - orange (not sure)

This can be done through the accounts management (for journals).

**Comment by lauradryad** [comment]
@ahamelers we are decided:

Priority - red
Flagged dataset - black [this will catch everything that isn't automatically flagged with the institution and submitter below]
Sensitive data - purple (HSI, endangered species, or anything else that needs a more thorough review)
Pending action - yellow (waiting for author, journal, cxore - any other party except dev - to proceed with dataset)
Institution - green (done through User page - to flag institutions like UCSB which need additional actions taken)
Submitter - blue (done through User page)
Technical - pink (JH: captures bugs, dev escalated issues, etc.; different from 'pending action flag' which might be used to capture escalations within the team or author reply, journal response, etc.)

**Comment by ahamelers** [comment]
@lauradryad So am I adding:

Flagged dataset - black 
Sensitive data - purple
Pending action - yellow
Technical - pink

?

Is there any better phrase than 'flagged dataset' for the first one? Just because these are all dataset flags we're talking about, I feel that the whole category of flagged datasets is 'flagged datasets' and so to name one of the flags 'flagged dataset' seems confusing. I mean, just try reading that sentence!

**Comment by lauradryad** [comment]
Yes to your list... re the name, I can't think of anything much better besides 'Other'. Special attention? Caution?

**Comment by jleighherzog** [comment]
ALL: For the 'integrated' flag, I was thinking it would be useful to know whether we should expect to see notifications of acceptance/rejection from the journals. At a glance, we could know whether it's safe to proceed to curate based on what's in the Activity Log. But, if you don't think that would be helpful, we can do without.
